documentation/library_guide/dynamic_selection_api/functions.rst

```diff
@@ -0,0 +1,595 @@
+Functions
+#########
+
+The dynamic selection API is an experimental feature in the |onedpl_long| 
+(|onedpl_short|) that selects an *execution resource* based on a chosen 
+*selection policy*. There are several functions provided as part 
+of the API.
+
+select
+------
+
+.. code:: cpp
+
+  namespace oneapi::dpl::experimental {
+    template<typename P, typename... Args> 
+    selection_t<P> select(P&& p, Args&&... args);
```

Comment:
Inconsistent way of describing function signatures. Somewhere we use `P` in others we use `Policy`. Somewhere we are even doing mistakes :)

Reply:
Addressed.